src/recommendation/src/main/python/TrainTestSplit.py

```diff
@@ -0,0 +1,211 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root for information.
+
+import numpy as np
+
+
+# Methods for train-test split.
+class TrainTestSplit:
+
+    @staticmethod
+    def min_rating_filter(self, min_rating, by_customer=True):
```

Comment:
Can you implement these in scala then just call into the scala methods to keep python only code to a minimum?